packages/react-components/src/date-input/src/DateRangeInput.tsx

```diff
@@ -137,7 +139,7 @@ export interface InnerDateRangeInputProps extends InteractionStatesProps {
     forwardedRef: ForwardedRef<any>;
 }
 
-const DateInput = forwardRef<any, "input">(({
+const DateInput = forwardRef<HTMLInputElement, any>(({
```

Comment:
What's the problem with "input" ? Is it because we do accept additional props like "validationState" and "onDateChange" ?